fpdcc/ccfp-asset-dashboard
# Migrate project categories model

## Overview
This PR:

- Closes #153 
- Closes #157


### Notes
We'll need to change the data in staging and production once this deploys. The migration change won't upset the existing data with the category/subcategory fields — it'll just move it all the new `ProjectCategory.name` field.

## Testing Instructions
- visit review app: https://ccfp-asset-d-feature-ca-yf8q7w.herokuapp.com
- to test the category model change:
  - visit the home page
    - make sure the projects list view works without a category
    - add a new project with the new categories
  - make sure the cip planner works without a category
